### DISCUSSION TITLE
anybody working on a Javascript decoder engine?

### DISCUSSION THREAD

**Discussion by mhaberler** [theengs/decoder#403] [Ideas]
device descriptors are already held in JSON format

so it would boil down to redoing decoder.cpp in Javascript

anybody working on this - drop me a note

-Michael

**Comment by 1technophile** [comment]
Decoder can be integrated to JS through Web Assembly also

**Reply by mhaberler** [reply]
I appreciate the suggestion, but that's a bit above my paygrade for now

could you point me to the code powering api.theengs.io which would be a good start for me?

thanks! 
Michael

**Reply by 1technophile** [reply]
I have done this integration (JS), but did not have the time to publish it yet
